src/circuit/v2/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const RELAYED_TAG = 'relayed'
+
+// circuit v2 connection limits
+// 2 min is the default connection duration
+export const DEFAULT_DURATION_LIMIT = 2 * 60 * 1000
+// 128k is the default data limit
+export const DEFAULT_DATA_LIMIT = 1 << 17
```

Comment:
constants copied from https://github.com/libp2p/go-libp2p/blob/master/p2p/protocol/circuitv2/relay/resources.go#L61-L66

Reply:
It's worth adding this as a comment in the code for people looking at it later